docs/docs/cmd/spfx/project/project-upgrade.md

```diff
@@ -85,3 +85,789 @@ Get instructions to upgrade the current SharePoint Framework project to the late
 ```sh
 m365 spfx project upgrade --output tour
 ```
+
+## Response
+
```

Comment:
Let's explain when the following output will be produced, eg. _When upgrading an SPFx project built using version x.y.z to SPFx version x.y.z you'll get output similar to following:_